indico/htdocs/sass/modules/_roombooking.scss

```diff
@@ -349,3 +349,9 @@ table.booking-stats {
         }
     }
 }
+
+#modify-room {
+    .subFieldWidth {
+        width: 118px;
```

Comment:
this feels like a dirty hack (especially adding that ID and that hardcoded size)... why not fix the code/css of the field. for example, you could use absolute positioning instead of a float to position the icon